src/lib/components/FilterForm.tsx

```diff
@@ -44,7 +56,7 @@ export const FilterForm: React.FC<IFilterFormProps> = () => {
   const setExpertsFilter = useCallback(
     _.debounce((checkedTypes) => {
```

Comment:
```suggestion
    _.debounce((checkedTypes: ICheckboxes) => {
```
use proper function typing instead of assertion to make sure function calls are correctly typed

Reply:
fixed